src/screen/MyIdentityScreen/RequestJudgement.tsx

```diff
@@ -40,7 +40,14 @@ function RequestJudgement({display, address, detail}: PropTypes) {
   const startTx = useCallback(
     async (index: number, fee: BN) => {
       if (api) {
-        await start(api, address, 'identity.requestJudgement', [index, fee]);
+        await start({
+          api,
+          address,
+          txMethod: 'identity.requestJudgement',
+          params: [index, fee],
+          title: `Sending transaction identity.requestJudgement`,
+          description: 'TODO',
```

Comment:
what's TODO?

Reply:
forgot to fix this, it's now `Request a judgement from a registrar.`